packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -403,25 +403,37 @@ export class TextArea
   };
 
   renderCharacterLimit = (): VNode => {
```

Comment:
Can you return null when there's no max length and update the return type accordingly?

Reply:
i think this needs to be update across the repo. Should we open an issue for this one?